database/ffldb/ff/flatfiledb.go

```diff
@@ -101,3 +103,29 @@ func (ffdb *FlatFileDB) store(storeName string) (*flatFileStore, error) {
 	}
 	return store, nil
 }
+
+// DeleteUpToLocation deletes the store data up to the given location file.
```

Comment:
Please update comment to something similar to what's written in the interface, explaining about the guarantees this gives